eng/pipelines/templates/steps/archetype-sdk-docs.yml

```diff
@@ -28,7 +28,7 @@ steps:
     workingDirectory: $(Build.BinariesDirectory)
     displayName: Download and Extract Required Software
 
-  - ${{ if ne(variables['Build.DefinitionName'], 'net - client - ci')}}:
+  - ${{ if and(ne(variables['Build.DefinitionName'], 'net - client - ci'), ne(variables['Build.DefinitionName'], 'net - client'))}}:
```

Comment:
I don't like that we are checking for specific definition names here. We should figure out another way to do this that scales properly. @mitchdenny any thoughts?

Reply:
Actually if since those pipelines will be deleted, I can just remove this check.

Reply:
We are getting rid of the ```net - client - ci``` and ```net - client``` pipelines anyway right?